docs/user_manual/guides/review_sign_offs.md

```diff
@@ -0,0 +1,20 @@
+# Review Case Sign-Offs
+
+When preparing a case for release, the reviewer must ensure that all sign offs have been completed.
```

Comment:
For consistency, sign offs --> sign-offs

Reply:
fixed